Makefile

```diff
@@ -144,6 +144,14 @@ operator-sdk:
 		echo "Using operator-sdk cached at $(OPERATOR_SDK)";\
 	fi
 
+.PHONY: create-performance-profile
+create-performance-profile:  build-output-dir
+	echo "Creating performance profile"
+	mkdir -p $(TOOLS_BIN_DIR); \
+	LDFLAGS="-s -w "; \
+	LDFLAGS+="-X github.com/openshift-kni/performance-addon-operators/cmd/performance-profile-creator "; \
```

Comment:
in a future PR, let's make sure we need these LDFLAGS. We copy stuff that works around too often (yes, I'm guilty of this too and it recently burned me of, exactly the reason why I'm raising this).

Reply:
Yep, I am guilty of copying from the other target:) Will double check and update in a subsequent PR.